### DISCUSSION TITLE
网站发布文章会自动插入多余区块

### DISCUSSION THREAD

**Discussion by zerobikappa** [Gamuxorg/bbs#216] [游戏或网站运行反馈]
浏览器：firefox 98.0.1
系统：5.16.15-arch1-1 x86_64 GNU/Linux
描述：
https://www.linuxgame.cn/parquet-wine
每次修改完发布就会自动插入一个轮播图片以及一个文章正文的区块，就算点击移除区块也没有用，只会越删越多。  

另外我不确定封面图是不是在「设置特色图片」那里上传，但是我点击「设置特色图片」然后上传图片的话，网站会提示「无法将上传的文件移动至wp-content/uploads。」

**Comment by hyh19962008** [comment]
因为很难修所以暂时不修复，把所有的分隔符都删除掉后再更新就可以了。
封面是特色图片，上传的问题修复了。

**Comment by zerobikappa** [comment]
> 因为很难修所以暂时不修复，把所有的分隔符都删除掉后再更新就可以了。 封面是特色图片，上传的问题修复了。

我尝试将所有分隔符（不只是多余的那些，而是所有的分隔符）全部删除，虽然不会越删越多了，但仍然会生成多余的分隔符。就在轮播的图片与文章正文的文字之间，多了一个「轮播图片」和「文章正文」的分隔符。是我有哪里做得不够吗？

**Comment by hyh19962008** [comment]
最上面最上面，也就是所有图片的上面，还有一个也要删掉

**Comment by zerobikappa** [comment]
> 最上面最上面，也就是所有图片的上面，还有一个也要删掉

那个有删啊，大标题与图片之间的「轮播图片」分隔符，还有图片与正文之间的几个分隔符，全部都有删的，不过还是会有这样的情况

**Comment by nukemiko** [comment]
我也碰上了一样的问题，“轮播图片”和“文章正文”两个分隔符根本删不完，即便是编辑空文章都会自动创建这些分隔符
